.circleci/config.yml

```
@@ -125,6 +128,12 @@ jobs:
               TARGET=sparkleverse
               RELEASE_STAGE=sparkleverse
             fi
+            if [ $CIRCLE_BRANCH = "sparkle1" ]; then
+              PREFIX=SPARKLE1_
+              ENV=prod
+              TARGET=sparkle1
+              RELEASE_STAGE=production
+            fi
```

I think this needs to be changed?
```suggestion
            if [ $CIRCLE_BRANCH = "sparkle1" ]; then
              PREFIX=SPARKLE1_
              ENV=sparkl1
              TARGET=sparkle1
              RELEASE_STAGE=sparkle1
            fi
```